app/Http/Requests/PostRequest.php

```diff
@@ -24,8 +24,8 @@ public function rules()
         return [
             'northstar_id' => 'required|string',
             'campaign_id' => 'required',
+            'quantity' => 'int|required',
```

Comment:
This shouldn't be required. It's a nullable field and campaigns going forward might not require it so we need to be flexible.

Reply:
ah yes!

Reply:
actually this is for v2 endpoints. But it should still be added and be `nullable|int`, right?

Reply:
yea we can leave it as not required for `v2`, since we are supporting both the new way of collecting quantity (if we have it) and the old way at the same time.